sdk/src/router/public/router-utils.ts

```diff
@@ -114,13 +117,13 @@ export class RouterUtils {
           addressesToLookup
         );
         try {
-          v0TxSize = await tx.txnSize({
+          v0TxSize = tx.txnSize({
             latestBlockhash: MEASUREMENT_BLOCKHASH,
             maxSupportedTransactionVersion: opts.maxSupportedTransactionVersion,
             lookupTableAccounts,
           });
 
-          if (v0TxSize !== undefined && v0TxSize < ENCODED_LIMIT) {
+          if (v0TxSize !== undefined && v0TxSize < TX_BASE64_ENCODED_SIZE_LIMIT) {
```

Comment:
As you know, the result of ``txnSize`` is now less than or equal to ``TX_SIZE_LIMIT``. 🙂